controller/space_iterations_test.go

```diff
@@ -468,7 +464,7 @@ func (rest *TestSpaceIterationREST) TestOnlySpaceOwnerCreateIteration() {
 		rootItr = &iteration.Iteration{
 			SpaceID:    newSpace.ID,
 			Name:       newSpace.Name,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c